### PR TITLE
Sukh submission fe

### PR DESCRIPTION
**Pull request by sukhpreetp**
Current UI (button needs to connect with home page): 
Image 15133

### CODE REVIEW

File: src/screens/auth/SubmissionConfirm.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            >
+              What can you do next?
+            </Text>
+            <Text
+            style={{
+              alignSelf: "flex-start",
+              paddingBottom: 5
+            }}
+            
+            ><Entypo name="dot-single" size={20} color="black" />Go inside but stay alert</Text>
```
Comment (TheerthaS) [inline src/screens/auth/SubmissionConfirm.js:136]: the dot points look a little off in the screenshot. Maybe you could see FlatList? https://reactnative.dev/docs/flatlist

File: src/navigation/AppNavigator.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ import Home from "../screens/Home";
 import SecondScreen from "../screens/SecondScreen";
 
 // Auth screens
-import Login from "../screens/auth/Login";
```
Comment (TheerthaS) [inline src/navigation/AppNavigator.js:12]: was there a reason you deleted this?

**Review by TheerthaS — CHANGES_REQUESTED** [review]

File: src/navigation/AppNavigator.js (hunk 2 above)
Comment (TheerthaS) [reply src/navigation/AppNavigator.js:12]: Also maybe we should move this screen out of /auth since it's not really auth stuff?

Comment (sukhpreetp) [reply src/navigation/AppNavigator.js]: That was just to view the UI. I have fixed this

File: src/screens/auth/SubmissionConfirm.js (hunk 1 above)
Comment (sukhpreetp) [reply src/screens/auth/SubmissionConfirm.js]: I have tried implementing that but to me it still looks the same (maybe I just can't see) but I think for now lets keep this and if we have time we can change it

**Review by TheerthaS — APPROVED** [review]

File: src/screens/auth/SubmissionConfirm.js (hunk 1 above)
Comment (TheerthaS) [reply src/screens/auth/SubmissionConfirm.js]: sounds good!

**Pull request merged**